src/uwtools/tests/drivers/test_forecast.py

```diff
@@ -167,6 +167,22 @@ def test_create_namelist_without_base_file(create_namelist_assets):
         assert out_file.read() == expected
 
 
+def test_forecast_run():
+    """
+    Tests that the forecast executable is built successfully with the user provided run commands.
+    """
+    hera_expected = "srun --export=ALL"
+    assert hera_expected == FV3Forecast.run("srun", "--export=ALL")
+
+    cheyenne_expected = "mpirun -np 4"
+    assert cheyenne_expected == FV3Forecast.run("mpirun", "-np", 4)
+
+    wcoss2_expected = "mpiexec -n 4 -ppn 8 --cpu-bind core -depth 2"
+    assert wcoss2_expected == FV3Forecast.run(
+        "mpiexec", "-n", 4, "-ppn", 8, "--cpu-bind", "core", "-depth", 2
+    )
```

Comment:
Do these tests work? `run()` isn't a static or class method, so I don't see how it can be called without first constructing an `FV3Forecast` instance. Maybe I'm missing something...

Reply:
I updated the tests so now they should be constructing an instance of `FV3Forecast` similar to other methods within the class.